### PR TITLE
back adopt_pack and checkout_contribution scripts

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: contribution/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This folder is used by contributors.
```
Comment (RotemAmit) [inline contribution/README.md:1]:
```suggestion
This folder is used for contributions related scripts.
```

**Review by kobymeir — APPROVED** [review]

**Review by kgal-pan — CHANGES_REQUESTED** [review]
Looks good overall! 

Just one thing that I believe need to be addressed:

- The file `git_pull_master_into_fork.sh` is also referenced in `.circleci/is_file_up_to_date.sh`:

```bash
❯ grep "git_pull_master_into_fork.sh" -rnw .
./.circleci/is_file_up_to_date.sh:34:          echo "Run ./.circleci/git_pull_master_into_fork.sh or merge manually from upstream demisto content"
```

So we probably want to change the path there as well.

**Comment by israelpoli** [comment]
I changed the message to contributions to the new location
although the script will still exist in the old location (in addition to the new location)

**Review by kgal-pan — APPROVED** [review]
Nice work!

**Pull request merged**